velox/vector/VectorPool.cpp

```diff
@@ -30,6 +30,9 @@ VectorPtr VectorPool::get(const TypePtr& type, vector_size_t size) {
 }
 
 bool VectorPool::release(VectorPtr& vector) {
+  if (FOLLY_UNLIKELY(vector == nullptr)) {
```

Comment:
This is nice as it makes it easier to use the API. Perhaps, update the method comments to clarify that it is safe to pass null.